cyber/event/perf_event.h

```diff
@@ -157,7 +158,7 @@ class TransportEvent : public EventBase {
  private:
   std::string adder_ = "";
   uint64_t msg_seq_ = 0;
-  uint64_t channel_id_ = UINT64_MAX;
+  uint64_t channel_id_ = std::numeric_limits<u_int64_t>::max();
```

Comment:
Better use the more wildly accepted `uint64_t` rather than `u_int64_t` defined in <sys/types.h>.

Reply:
Agree. And at least the two `T` in `T channel_id_ = std::numeric_limits<T>::max();` should be aligned.